Dockerfile

```diff
@@ -11,7 +11,9 @@ ENV IS_DOCKER 1
 RUN mkdir -p /pip/{cache,build}
 ADD requirements /pip/requirements
 WORKDIR /pip
-RUN pip install -b /pip/build --download-cache /pip/cache --no-deps -r /pip/requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/
+# Download this securely from pyprepo first.
+RUN pip install --no-deps --find-links https://pyrepo.addons.mozilla.org/ peep
```

Comment:
could you put ``peep>=X.X`` so that if it breaks something we at least know the last known working version?